population/create_glpi_computer_redfish.py

```diff
@@ -156,6 +179,12 @@ def main() -> None:
     sku = args.sku
     switch_config = args.switch_config
     no_verify = args.no_verify
+    sunbird_username = args.sunbird_username
+    sunbird_password = args.sunbird_password
+    if args.sunbird_url:
+        sunbird_url = validate_url(args.sunbird_url)
+    else:
+        sunbird_url = args.sunbird_url
```

Comment:
So you're just setting this to None if there is no sunbird args passed? Maybe setting to None would be more explicit?

Reply:
Good idea - addressed in latest commit